haskell-xmpp.cabal

```diff
@@ -35,8 +35,9 @@ flag debug
   default: False
 
 library
+  default-language: Haskell2010
   Hs-Source-Dirs: ./src
-  Build-Depends: base
+  Build-Depends: base < 5
```

Comment:
It is better to add the lower bound as well to indicate supported versions of GHC

Reply:
It's not a pleasant experience having upgrades failing because of cabal files being to restrictive, I'd argue that it's better to be lax and have compile errors than strict and do busy work.

Reply:
If you want to support older versions of GHCs as well, then you should test it. It is the responsibility of the maintainers to keep track of that. The lack of restriction is troublesome as well.
You specified above that it is tested with `GHC-8.8.3` only, then it should reflect it in the base version, to be clear about what this package supports.

Reply:
You're right we should test it, It just produces so much extra work to keep on upgrading these bounds. (must be like 50% of the time I spend on upgrading compiler/nixpkgs).